src/sage_docbuild/conf.py

```diff
@@ -374,7 +374,7 @@ def set_intersphinx_mappings(app, config):
 if os.environ.get('SAGE_USE_CDNS', 'no') == 'yes':
     mathjax_path = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"
 else:
```

Comment:
An "easy" fix would be to cut the last path component of `MATHJAX_DIR` and prefix it to `mathjax_path`.

Reply:
I mean the lines L377,378.

Reply:
No. This would not work...